.github/ISSUE_TEMPLATE.md

```diff
@@ -1,21 +1,18 @@
-Bug Report
-==========
-Check to see if a bug report already exists.  If you still want to create a new issue please include the information below.
+<!-- This form is for bug reports and feature requests only.
+
+If you're looking for help check [Discord](https://discord.gg/sia) and the Sia [website](sia.tech).
+
+Please make sure the bug report doesn't already exist in the GitHub issues.
+-->
+
+# Is this a BUG REPORT or FEATURE REQUEST?
```

Comment:
most people probably won't delete this line, and it's huge. It's probably sufficient to say `If this is a feature request, please put "Feature Request:" in the issue title` in the comment.